src/modules/TeamsList/components/Teams/index.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { FC } from 'react';
+import { PageTitle } from 'typography';
+
+import { TeamsHeader } from './components/TeamsHeader';
+import { MyTeam } from './components/MyTeam';
+import { Team, TeamList } from 'types';
+import { TeamItem } from './components/TeamItem';
+
+type TeamsProps = {
+  teams: TeamList;
+  title?: string;
+  myTeam?: Team;
+  userId: string;
+};
+
+export const Teams: FC<TeamsProps> = ({ title, teams, myTeam, userId }) => (
+  <>
+    <PageTitle>{title || 'Teams'}</PageTitle>
+    {myTeam ? <MyTeam team={myTeam} userId={userId} /> : <TeamsHeader />}
+    {teams.count &&
+      teams.results.map((team, index) => (
+        <TeamItem
+          key={index}
```

Comment:
Key must be unique. It's bad practice to put the index in the key :)

Reply:
done.
Should the key be unique on the page or in just a list?